src/Service.Tests/SqlTests/RestApiTests/Put/PutApiTestBase.cs

```diff
@@ -412,6 +412,53 @@ public virtual async Task PutOne_Nulled_Test()
                 );
         }
 
+        /// <summary>
+        /// Test to validate successful execution of a request when a computed field is missing from the request body.
+        /// In such a case, we don't attempt to NULL out computed field (as per PUT semantics) but instead skip updating/inserting the field. 
+        /// </summary>
+        [TestMethod]
+        public virtual async Task PutOneWithComputedFieldMissingFromRequestBody()
+        {
+            // Validate successful execution of a PUT update when a computed field (here 'last_sold_on_update')
+            // is missing from the request body.
+            string requestBody = @"
+            {
+                ""book_name"": ""New book"",
+                ""copies_sold"": 101
+            }";
+            string expectedLocationHeader = $"id/1";
+
+            await SetupAndRunRestApiTest(
+                    primaryKeyRoute: expectedLocationHeader,
+                    queryString: null,
+                    entityNameOrPath: _entityWithReadOnlyFields,
+                    sqlQuery: GetQuery("PutOneUpdateWithComputedFieldMissingFromRequestBody"),
+                    operationType: EntityActionOperation.Upsert,
+                    requestBody: requestBody,
+                    expectedStatusCode: HttpStatusCode.OK
+                );
+
```

Comment:
The query `PutOneUpdateWithComputedFieldMissingFromRequestBody` doesn't actually verify that the computed field `last_sold_on_update` isn't touched (Nulled out because of exclusion from request body/ PUT semaintics.)

In your PR description, you state  _"When a read-only field is excluded from the request body for mutation, we won't NULL out the field's value but let the responsibility on the database to deal with it"_. When you say _"let the responsibility on the database to deal with it"_, is that what is happening here? the database would return an error if DAB erroneously included the computed field `last_sold_on_update` in the update query (even when user excludes the field from the GraphQL/REST body)?

Reply:
> When you say "let the responsibility on the database to deal with it", is that what is happening here? the database would return an error if DAB erroneously included the computed field last_sold_on_update in the update query (even when user excludes the field from the GraphQL/REST body)?

Correct. The thing that the PUT request executed successfully on a table with read-only field confirms that we did not attempt to update the value of the read-only field.

Reply:
I'll add to the comments.